src/libraries/System.DirectoryServices/src/System/DirectoryServices/ActiveDirectory/ActiveDirectoryPartition.cs

```diff
@@ -6,9 +6,9 @@ namespace System.DirectoryServices.ActiveDirectory
     public abstract class ActiveDirectoryPartition : IDisposable
     {
         private bool _disposed;
-        internal string partitionName;
-        internal DirectoryContext context;
-        internal DirectoryEntryManager directoryEntryMgr;
+        internal string partitionName = null!;
```

Comment:
Yes and no:
- this is an abstract class and all derived classes here make sure this is assigned (see below)
- internal ctor has artifically non-null name because nothing is passing nullable
- protected constructor will make this value always null but it has 0 references inside of S.DirectoryServices and everyone outside won't see this field because it's protected
- more than one reference of partitionName will throw NRE if it's null

based on that IMO this should remain non-nullable